src/pages/ProfileSettings.tsx

```diff
@@ -1,20 +1,261 @@
-import BillingCard from "@/templates/BillingCard";
-import ProfileCard from "@/templates/ProfileCard";
+import { Card } from "@/components/ui/card";
+import { useState } from "react";
+import { UserRound } from "lucide-react";
+import LabelWithText from "@/components/elements/LabelWithText";
+import { Button } from "@/components/ui/button";
+import { X } from "lucide-react";
+import InputWithLabel from "@/components/elements/InputWithLabel";
+import { Pencil } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import InputWithTagLeft from "@/components/elements/InputWithTagLeft";
 
 export default function ProfileSettings() {
+  const [image, setImage] = useState(null);
+
+  const onImageChange = (event: React.FormEvent<HTMLInputElement>) => {
+    if (event.currentTarget.files && event.currentTarget.files[0]) {
```

Comment:
you can add a variable before this line `const file = event.currentTarget.files ` and use it